package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "jhipster-bot",
+  "version": "0.0.0",
+  "description": "Enable you to generate a JHipster App using Slack",
+  "main": "jhipster-bot.js",
+  "scripts": {
+    "test": "mocha test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jhipster/jhipster-bot.git"
+  },
+  "author": "CarlKlagba",
+  "license": "BSD-2-Clause",
```

Comment:
It was agreed in the mailing-list that all projects should be on the Apache 2 license: https://groups.google.com/forum/#!topic/jhipster-dev/7oRvbr6bwkU
Of course if you don't agree with that, you can discuss it on the mailing list. Moreover this repository already has a license.txt file set by Julien.